wagtail/admin/templates/wagtailadmin/pages/index.html

```diff
@@ -2,7 +2,7 @@
 {% load wagtailadmin_tags i18n %}
 
 {% block header %}
-    {% include 'wagtailadmin/shared/headers/slim_header.html' with title=header_title search_url=index_results_url %}
+    {% include 'wagtailadmin/shared/headers/slim_header.html' with title=header_title search_url=index_results_url buttons=header_buttons %}
```

Comment:
While we're here, let's add icon support for the listing view as well

```suggestion
    {% include 'wagtailadmin/shared/headers/slim_header.html' with title=header_title icon_name=header_icon search_url=index_results_url buttons=header_buttons %}
```